bin/mainflux-launch.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+###
+# Launches all Mainflux Go binaries when they are installed globally.
+# Also launches NATS broker instance, expecting that
+# `gnatsd` is installed globally.
+#
+# Expects that Cassandra is already installed and running.
+#
+# Does not launch NodeJS MQTT service - this one must be launched by hand for now.
+###
+
+# Kill all mainflux-* stuff
+function cleanup {
+	pkill mainflux
+}
+
+gnatsd &
+# Wait a bit for NATS to be on
+sleep 0.1
+mainflux-http &
```

Comment:
The services should be built as well.

Reply:
This is developer helper script.

Maybe we should move it from the `bin` to `utils` or `helpers` or `scripts` or something?

Reply:
The main problem I have with this script is that it is useless. You define it as developer helper script. If we start from that premise, we're faced with the fact that:
- no infrastructure has been configured (NATS, PostgreSQL, nginx)
- no configuration can be provided to the services

While we can say "it is up to the developers to setup their infrastructure", the default configuration used by the services (since no other can be provided) is quite opinionated (e.g. it expects PostgreSQL to be available at `localhost:5432` which might not always be the case.